lib/crc-model-visitor.js

```diff
@@ -3,31 +3,54 @@ const _ = require('lodash');
 
 (function () {
 
+    const defaultDeclarators = [
+        'FunctionDeclaration',
+        //'FunctionExpression',
+        //'ArrowFunctionExpression',
+        'ClassDeclaration',
+        'ClassExpression',
+        'VariableDeclaration',
+        'VariableDeclarator'
+    ];
+
+    let _customDeclarators = defaultDeclarators;
+
     /**
      * Collects information needed to construct an overall CRC Model
      * by "visiting" the abstract syntax tree.
      */
     class CrcModelVisitor {
 
         /**
+         *
+         * @static
+         * @member {CrcModelVisitor}
+         * @property {Array<string>} defaultDeclarators - The default set of node types to be used to identify classes.
+         *
+         */
+        static get defaultDeclarators() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide a setter matching this getter for 'defaultDeclarators'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2376)